public_api.ts

```diff
@@ -56,6 +56,7 @@ export { AppTableHeaderComponent } from './src/app/modules/table/table-header/ta
 export { AppTableRowComponent } from './src/app/modules/table/table-row/table-row.component'
 export { AppButtonComponent } from './src/app/modules/button/button.component';
 export { FileUploadDragDropDirective } from './src/app/directives/file-upload-drag-drop.directive';
+export { DataTestDirective } from './src/app/directives/data-test.directive'
```

Comment:
```suggestion
export { DataTestDirective } from './src/app/directives/data-test.directive';
```

Reply:
Oh my word. Nice catch, and thank you so much 🤦